package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "ramsay-ui",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/ramsayinnovations/ramsay-ui.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/ramsayinnovations/ramsay-ui/issues"
+  },
+  "homepage": "https://github.com/ramsayinnovations/ramsay-ui#readme",
+  "dependencies": {
+    "@emotion/css": "^11.1.3",
+    "@emotion/react": "^11.1.4",
+    "@emotion/server": "^11.0.0",
+    "@emotion/styled": "^11.0.0",
+    "@headlessui/react": "^0.2.0",
+    "autoprefixer": "^10.2.1",
+    "postcss": "^8.2.4",
+    "postcss-preset-env": "^6.7.0",
+    "react": "^17.0.1",
+    "react-dom": "^17.0.1",
+    "react-hook-form": "^6.14.2",
+    "react-icons": "^4.1.0",
+    "react-scroll": "^1.8.1",
```

Comment:
Please remove `react-scroll`.  We don't need it for `ramsay-ui` (but keep it for `ramsay-site`

Reply:
fixed